src/v2/guide/render-function.md

```diff
@@ -536,6 +536,7 @@ Everything the component needs is passed through `context`, which is an object c
 - `props`: An object of the provided props
 - `children`: An array of the VNode children
 - `slots`: A function returning a slots object
+- `scopedSlots`: (2.6.0+) An object that exposes passed-in scoped slots
```

Comment:
Shall we add note about this includes `slots` as functions?

Reply:
yes